### ISSUE TITLE
No details regarding the Custom Model training.

### ISSUE THREAD

**Issue by lakpa-tamang9** [OODRobustBench/OODRobustBench#1]
Hi, Thank you for your amazing work.

It seems that there is error in the hyperlink where it says 
Custom models were trained by ourselves, see [here](https://github.com/oodrobustbench/oodrobustbench?tab=readme-ov-file#5.-Custom-Models) for details.

The link does not open.

**Comment by TreeLLi** [comment]
Hi, 

thanks much for your interest in our work. The custom models were trained by ourselves. I am not sure which error you are referring to.

**Comment by lakpa-tamang9** [comment]
Hi Thanks for your reply.

I know you trained the custom models. In the readme, you have provided the link for details of training custom models. The link does not open. Please see the reference below.

I wanted to know the details of the training.

![Image](https://github.com/user-attachments/assets/bc3ca268-e037-4afe-800e-94f1d619ec2f)

**Comment by TreeLLi** [comment]
I see. That is just a cross-refernece to the section 5 in the same page which I believe you have already read. I will fix this bug. Regarding the details of training, the values of most hyper-parameters used are written in the file name of the pre-trained weights. For example, given a weight file named `custom_convmixer_trades_trades_seed0_bs512_lr0.1_wd0.0001_sgd_50ep_eps0.5_beta0.1`, the hyperparameters are separted by `_` and valued in order:

1. custom: unified ID, just ignore it.
2. convmixer: model architecture
3. trades: adversarial training method
4. trades: training loss
5. seed0: random seed
6. bs512: batch size 512
7. lr0.1: learning rate of 0.1
8. wd0.0001: weight decay 1e-4
9. sgd: optimizer
10. 50ep: 50 epochs
11. eps0.5: eps 0.5
12. beta0.1: hyperparameter used in trades.

 You can find more details in appendix C.2 in our paper.

**Comment by lakpa-tamang9** [comment]
Thank you for your clarification.
